.env.docker.production.example

```
@@ -50,7 +50,7 @@ AUTH_GITHUB_SCOPE=read:user,user:email,read:org
 AUTH_GITHUB_FIRST_NAME_KEY=name
 AUTH_GITHUB_LAST_NAME_KEY=login
 AUTH_GITHUB_IDENTIFIER_KEY=id
-AUTH_GITHUB_REDIRECT_ALLOW_LIST=https://globalping.io,https://dash.globalping.io,https://auth.globalping.io,https://staging.globalping.io,http://localhost:13000
+AUTH_GITHUB_REDIRECT_ALLOW_LIST=https://globalping.io,https://globalping.io/auth/callback,https://dash.globalping.io,https://dash.globalping.io/auth/callback,http://localhost:13000,http://localhost:13000/auth/callback,https://auth.globalping.io/oauth/authorize
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_GITHUB_REDIRECT_ALLOW_LIST key should go before the AUTH_GITHUB_SCOPE key

(UnorderedKey)